Review the current proposal of fundraising Working Group (#14)
django/dsf-working-groups: active/fundraising.md

## Scope of responsibilities

This working group coordinates fundraising efforts, particularly around corporate and major donations.

The goals of the group are:

> - Acquire funding to secure the financial future of Django and support future growth of Django

Reviewer: I think this reads fine, but could also be:
```suggestion
- Acquire funding to secure the financial future of Django and support its future growth
```

> - Build and strengthen relationships between Django and donor sources

Reviewer 2: ```suggestion
- Build and strengthen relationships between Django and funding sources
```

Seems to me like we might as well not just think of donors here?

Reviewer 3: I tend to agree - for example I'd like to think about more broadly going after grants (eg I think we have a good shot at raising money from the [Sovereign Tech Fund](https://www.sovereigntechfund.de/) so thinking more broadly than "donations" seems great to me

> - Raise awareness of the Django Web Framework and its community with the broader world

Reviewer 2: With this one, I’m struggling to see why this is a responsibility of the fundraising group? Raising awareness feels like something some of the other groups could be doing.

Reviewer 4: I think a natural byproduct of fundraising is raising awareness. It's not the primary goal, but it should be one of them. A sponsor who gives $10,000 without saying anything about Django is not the same as a sponsor who gives $10,000 and tells their communities about the donation and Django. There's a broader picture as to what fundraising could do. Making it clear to the working group that their efforts mean more than a the bottom line could help influence hard decisions.

Reviewer: Visibility also attracts more funding, so I also see value in this.

Reviewer 2: Sounds fair!

> - Raise awareness of the Django Web Framework and its community with the broader world

Reviewer: I don't know if "Web Framework" should be capitalised unless we use this phrasing elsewhere.

Delegated responsibilities:

> - Contacting, developing and implementing financial sponsorships with donor sources
> - Clarifying and elaborating on the various creative strategies that donor sources of different sizes and types can use to support Django

Reviewer 2: Rewording so it’s not just about donations (unless you do intend for it to be). I wonder if it’d be worth even being more specific and calling out different types of funding directly.

```suggestion
- Contacting, developing and implementing financial sponsorships with funding sources
- Clarifying and elaborating on the various creative strategies that funding sources of different sizes and types can use to support Django
```

## Initial membership

- Chair: TBD
- Co-Chair: TBD

> - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD

Reviewer 3: ```suggestion
- Board Liaison: Jacob Kaplan-Moss
```

I think this'll be (subject to board approval)

- Other members:
  - Jacob Kaplan-Moss
  - Priya Pahwa
  - Thibaud Colas
  - Tom Carrick
  - Sarah Abderemane
  - Stephanie Goulet

> 

Reviewer 5: May I be added too please?

## Future membership

Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.

This committee is currently forming and seeking volunteers, including chair and co-chair. 

### Navigating Conflicts of Interest

> Each new member must disclose any potential conflicts of interest. These disclosures will be made transparent by means of ____. 

Author: I'm unsure what is the best way, initially I thought about an email google group but could be something else

Reviewer 2: I’d expect it’d be more useful to disclose conflicts of interest as part of decision-making on a case by case basis, than upfront?

I’m not sure I see why this needs to be part of the WG charter to start with, it’s important but even for the board I don’t believe we have this so formally defined.

Reviewer 3: Yeah, I do think it's fine to leave the specific mechanism of COI disclosure out of the charter -- it can be an operating procedure, not something in the charter. 

That said, I think it might be sufficient to just ask for COI disclosures as part of the application process, in which case you can easily just add that there. Something like this could work:

> When new members self-nominate, they should disclose any potential COIs to the WG. They should also disclose any COIs that might affect a vote before that vote

Any person with a conflict of interest will need to recuse themselves from any votes or other decision-making related to their conflict.

> No more than X% of the Fundraising Working Group may contain individuals associated with the same organization

Author: I think this decision needs to be made by a group of people, not just me giving a number.

Reviewer 2: I don’t understand why this would be enough of an issue to call out here?

Reviewer 4: I think it's fair to be forward thinking about what this may turn into. Some of the larger employers are going to be ideal funding sources. It seems weird to permit the WG the possibility to have a voting bloc for that company.

I think it's wiser to take a more conservative stance from the start. It can be scaled back if it becomes a problem in the future. However, the reverse situation seems messier to untangle.

Reviewer 2: I can see the problem but am not convinced it’s worth having to define this in that clear-cut of a way. Would rather replace this with a line in "future membership" about having individuals from a good diversity of different organizations perhaps. 

The reverse situation – if it comes to that, the board can always decide to shut down a working group (requires a board vote).

Reviewer 4: Could you elaborate on what problems including this would invite?

Reviewer 2: Well first we have to agree on a X% figure, which to me feels like a distraction compared to other aspects of getting this group up and running. Then I see it being a distraction in the long term too when selecting new members.

For example, right now 50% of the group’s draft membership is associated with Djangonaut Space. If we’re concerned about voting bloc then that’s a problem at least the way that rule is written right now? It seems entirely plausible the group’s membership could have 50% DEFNA members, or Django Girls, or Wagtail; and consider adding one more. All of which wouldn’t be ideal but if it’s the right person at the right time, I’d rather not have to think about a hard and fast "X%".

Reviewer 4: Thank you Thibaud, that helps me understand your perspective. You're right that the way it's written now would be problematic with the proposed members. That needs a change. Though since I'm not a proposed member and one of your concerns is that this is distracting, I'll leave it to the remainder members to voice their opinions.

> Each new member must disclose any potential conflicts of interest. These disclosures will be made transparent by means of ____. 
> 
> Any person with a conflict of interest will need to recuse themselves from any votes or other decision-making related to their conflict.
> 
> No more than X% of the Fundraising Working Group may contain individuals associated with the same organization

Reviewer 6: A revision of the CIO section based on the above two comment threads, my opinion, and reading the DSF COI policy:
```suggestion
Members must disclose possible conflicts of interest (COI) to the Fundraising Working Group when they first self-nominate as well as if new COIs arise. No more than two members of the Fundraising Working Group may receive compensation from the same organization at any given time. 

In the spirit of aligning with the [Django Software Foundation Conflict of Interest Policy](https://www.djangoproject.com/foundation/conflict-of-interest/), members must disclose possible COIs that might affect a vote before that vote. That member can then choose to either recuse themselves from the vote or disclose all relevant facts regarding the COI. If they disclose all relevant facts, the remaining members will majority vote whether it will constitute a COI without them present. Ties will be brought to the board to decide prior to vote. The resolution of each COI must be recorded in the Fundraising Working Group notes.
```
My main question is: **do members of the Fundraising Working Group legally counted under the definition of an "Interested Person" from the official DSF COI Policy?** If they do, this section should explicitly state that we defer to that policy (and should also probably be simplified to mainly just say something like "Operates under the DSF COI Policy" with a link to that policy page).

Reviewer 2: Well, first time I hear of this policy! That’s certainly handy to have that all written down 😄 Yes, if this policy is under active use, to me it’s quite clear that a member of a working group is a "member of a committee with governing board delegated powers".

And with that in mind, I’d prefer we defer to that policy without any re-interpretation.

## Budget

> No allocated funds.

Reviewer: I guess it doesn't take money to make money after all.

Reviewer 6: I also think asking the Board for funds as needed is fine for now

## Comms

- Collaborative documents

> - Private channel in DSF slack

Reviewer: Not in love with Slack but it is already there. "Slack" should probably be capitalised.

Reviewer 6: I don't prefer Slack either (my preference would be private Django Forum thread if possible or email). 

I don't want a discussion on the specific communication method to hold up moving the charter forward though — I imagine anything like this can be adjusted pretty easily once we meet and determine what communication methods everyone prefers (aka: I'd use Slack if it makes the most sense for the group).

Author: @thibaudcolas @priyapahwa @jacobian @cgl any opinion?

> - At least monthly synchronous meetings

Reviewer: I think it's good to set a specific cadence rather than "at least monthly". I think just specifying monthly meetings doesn't exclude ad-hoc meetings from happening when needed or useful.

Reviewer 6: I agree

## Reporting

> We'll email a written report to the board every quarter.

Reviewer 2: I’d expect more regular updates, and most likely a shared dashboard with active fundraising leads or similar CRM-type data. This is quite critical information for the board to be aware of and have a chance to discuss as part of every meeting.

Reviewer 3: You could copy some language from what I used in #15 -- https://github.com/django/dsf-working-groups/blob/bde7b66cd4204e22a1b7418aa91fb7c5da2c0cb3/active/fellowship.md?plain=1#L60-L68

> We'll email a written report to the board every quarter.

Reviewer 6: I like the language Jacob quoted. Took a first stab at customizing it to the FWG below. I kept it high level as I imagine we'd establish a template report outline anyway and update it as needed to include everything the Board needs and wants to know.  

```suggestion
The Fundraising Working Group is required to submit regular reports on fundraising efforts. 
  
Frequency: at least quarterly, with a strong preference for some sort of live "dashboard" (read: Google Sheet) that's updated more regularly for the Board to check any time. 
  
The specifics of what needs to be in the report will change over time and should be an ongoing conversation between the Fundraising Working Group and the Board. It can and should change and evolve over time. It should probably should include things like anticipated next steps on funding opportunities, planned interactions with major donor leads, updates of other activities the group is working on, etc.
 
The Fundraising Working Group is also expected to report any meta-issues or problems to the board. E.g. ongoing strategic questions without answers, problems with DSF infrastructure, emerging issues, and so forth. 
```

PS: I left in the phrase "problems with DSF infrastructure" even though I'm not sure I actually know what kind of thing that is supposed to be describing.

Reviewer 2: Looks good to me, I’d prefer we aim for "at least monthly" as the work of this group is much more time-dependent.

## Scope of responsibilities

This working group coordinates fundraising efforts, particularly around corporate and major donations.

The goals of the group are:

- Acquire funding to secure the financial future of Django and support its future growth
- Build and strengthen relationships between Django and funding sources
- Raise awareness of the Django web framework and its community with the broader world

Delegated responsibilities:

- Contacting, developing and implementing financial sponsorships with funding sources
- Clarifying and elaborating on the various creative strategies that funding sources of different sizes and types can use to support Django

## Initial membership

- Chair: TBD
- Co-Chair: TBD
- Board Liaison: Jacob Kaplan-Moss
- Other members:
  - Priya Pahwa
  - Thibaud Colas
  - Tom Carrick
  - Sarah Abderemane
  - Stephanie Goulet

## Future membership

Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.

This committee is currently forming and seeking volunteers, including chair and co-chair. 

### Navigating Conflicts of Interest

> Members must disclose possible conflicts of interest (COI) to the Fundraising Working Group when they first self-nominate as well as if new COIs arise. No more than two members of the Fundraising Working Group may receive compensation from the same organization at any given time. 
> 
> In the spirit of aligning with the [Django Software Foundation Conflict of Interest Policy](https://www.djangoproject.com/foundation/conflict-of-interest/), members must disclose possible COIs that might affect a vote before that vote. That member can then choose to either recuse themselves from the vote or disclose all relevant facts regarding the COI. If they disclose all relevant facts, the remaining members will majority vote whether it will constitute a COI without them present. Ties will be brought to the board to decide prior to vote. The resolution of each COI must be recorded in the Fundraising Working Group notes.
> 

Reviewer 6: ```suggestion
The Fundraising Working Group adheres to the [Django Software Foundation Conflict of Interest Policy](https://www.djangoproject.com/foundation/conflict-of-interest/) and follows all procedures outlined within it, including those related to disclosure, reporting, and resolution of conflicts of interest. 
```
